test/models/friend_request_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+
+class FriendRequestTest < ActiveSupport::TestCase
+  def setup
+    @requester = users(:musa)
+    @requested = users(:kingsley)
+    @friend_request = FriendRequest.new(requester_id: @requester.id,\
+                                        requested_id: @requested.id)
+  end
+
+  test 'should be valid' do
```

Comment:
We can safely remove these validation tests as we can also remove their corresponding code in the model (see my comments in the request model). I realized this just now that the associations themselves act as validators. Therefore, your association tests using shoulda already took care of these validations